tycho/tests/routes/test_event.py

```diff
@@ -245,7 +245,7 @@
         assert (retrieved_event.to_primitive()
                 == retrieved_event2.to_primitive())
         if log:
-            mock_log.info.assert_called_with(event.to_primitive())
+            mock_log.info.assert_called_with(json.dumps(event.to_primitive()))
```

Comment:
one thought is maybe finding a more robust test to verify this behavior. Proof in the putting would be a log line emitted. Caplog fixture maybe? https://docs.pytest.org/en/latest/logging.html#caplog-fixture